students/AndyKwok/lesson02/GridPrinter.py

```diff
@@ -0,0 +1,39 @@
+#Description: Program that prints grids 
+#Author: Andy Kwok
+#Last Updated: 07/01/2018
+#ChangeLog: 
+#			v1.0 - Initialization
+
+
+#Function that prints a 2x2 grid
+def print_grid_basic():
+	for j in range(2):	
+		print('+' + ' - '*4 + '+' +  ' - '*4 + '+')
+		for i in range(4):	
+			print('|' + '   '*4 + '|' +  '   '*4 + '|')	
+	print('+' + ' - '*4 + '+' +  ' - '*4 + '+')
+	
+#Function that prints a 2x2 grid with various size per input value (grid size increase by value)
+def print_grid_one(x):
+	y = int(x/2)
+	for j in range(2):	
+		print('+' + ' - '*y + '+' +  ' - '*y + '+')
+		for i in range(y):	
+			print('|' + '   '*y + '|' +  '   '*y + '|')	
+	print('+' + ' - '*y + '+' +  ' - '*y + '+')
+
+	
+#Function that prints a nxn grid with various size per input values
+#First parameter is number of squares in each row/column, n
+#Second parameter is the width of each square 
+def print_grid_multi(x,y):
+	for j in range(x):	
+		for k in range(x):
+			print('+' + ' - '*y , end=' ')
+		print('+')
+		for i in range(x):	
+				print('|' + '   '*y , end=' ')
+		print('|')
+	for k in range(x):
+		print('+' + ' - '*y , end=' ')
+	print('+')
```

Comment:
generally good idea to avoid one letter vars; it is ok in certain cases like comprehensions (we'll learn later) but it is best to use meaningful vars so that the reader can easily understand the code.

also, if you would like to describe the function you can use docstrings, here is the format:
```
def print_grid(x, y):
    """Prints a grid with x by y size (etc)
    x: int
    y: int
    """
    pass 
```
there are various formats on how you define params and return .. but as long as you stick to triple double quotes, you should be good style wise.